usr/scripts/detectionOn.sh

```diff
@@ -0,0 +1,57 @@
+#!/bin/sh
+
+# Source your custom motion configurations
+. /usr/config/motion.conf
```

Comment:
Can we agree on /etc/motion.conf there?

Reply:
Sure ...
I was not sure how far you are on porting the scripts